.github/CODEOWNERS

```diff
@@ -96,6 +96,7 @@ packages/app-check-interop-types @hsubox76 @firebase/jssdk-global-approvers
 # Documentation Changes
 packages/firebase/index.d.ts @egilmorez @firebase/jssdk-global-approvers
 scripts/docgen/content-sources/ @egilmorez @firebase/jssdk-global-approvers
+docs-devsite/ @egilmorez @markarndt
```

Comment:
I don't think Mark has write access to this repo? cc: @markarndt

Reply:
I've added him, let me know if any other tech writers ought to be added.